lib/config/recommended-gts.js

```diff
@@ -1,4 +1,29 @@
 const base = require('./base');
 const gtsRules = require('../recommended-rules-gts');
 
-module.exports = [...base, { rules: gtsRules }];
+// see https://github.com/typescript-eslint/typescript-eslint/issues/8607
+// https://github.com/typescript-eslint/typescript-eslint/blob/v7.4.0/packages/eslint-plugin/src/configs/eslint-recommended-raw.ts
```

Comment:
why don't we use this config? because we can't force v7?
and we don't want a peer requirement? (I think that's what we ended up deciding on elsewhere, yeah?)

Reply:
That cannot be used directly. It's a utility function. Eslint needs an object as default export